src/Hamiltonians/ReducedDensityMatrix.jl

```diff
@@ -0,0 +1,104 @@
+"""
+    SingleParticleExcitation(i, j) <: AbstractHamiltonian
+
+Represent {i,j} element of the single-particle reduced density matrix:
+
+```math
+\\hat{ρ}^{(1)}_{i,j} = \\hat a^†_{i} \\hat a_{j}
+```
+
+where ``i`` and ``j`` are the `mode`
+
+# See also
+
+* [`single_particle_density`](@ref)
+* [`SingleParticleDensity`](@ref)
+* [`TwoParticleExcitation`](@ref)
+"""
+struct SingleParticleExcitation{I,J} <: AbstractHamiltonian{Float64}
+end
+
+SingleParticleExcitation(I::Int,J::Int) = SingleParticleExcitation{I,J}()
+
+function Base.show(io::IO, spd::SingleParticleExcitation{I,J}) where {I,J}
+    print(io, "SingleParticleExcitation($(I), $(J))")
+end
+
+LOStructure(::Type{T}) where T<:SingleParticleExcitation = AdjointUnknown()
```

Comment:
```suggestion
LOStructure(::Type{<:SingleParticleExcitation}) = AdjointUnknown()
```
Just a more compact way of writing this.